debian/control

```diff
@@ -19,7 +19,6 @@ Depends:
     molly-guard,
     mutt,
     ntp,
-    postfix,
```

Comment:
Why removing postfix? We configure it through templating https://github.com/wazo-platform/xivo-config/tree/master/templates/mail/etc/postfix
Maybe we should move it in xivo-config

Reply:
it was making the installation of the package fail. I have no idea why.

Reply:
Yeah maybe a conflict with the wazo-auth dependency: `mail-transport-agent`. Can you restart the job with the machine frozen and it will be easier to debug it

Reply:
could we make wazo-auth depend on postfix instead?

Reply:
* It's only a supposition (I'm not even sure that is related)
* No, because we don't (and won't) know which mail client user will used

Reply:
this is fixed by using aptitude like ansible is doing.